src/voxel.h

```diff
@@ -569,6 +580,8 @@ class VoxelManipulator /*: public NodeContainer*/
 	*/
 	u8 *m_flags;
 
+	static const MapNode ContentIgnoreNode;
```

Comment:
It's possible, but the function doesn't check for out-of-bounds indexing, which means I'd be making the functions less safe.

All it takes is one person who "thinks" they know what the functions do to rely on their old behaviour, and we have a crash (and if we're unlucky, an uncommon one).